Description: This patch update the
 location of docbookx.dtd and the
 place where the high scores file
 is stored.
Forwarded: not-needed
Author: Pierre Machard <pmachard@debian.org>
Last-Update: 2010-04-16

--- a/greed.xml
+++ b/greed.xml
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
 <!DOCTYPE refentry PUBLIC 
    "-//OASIS//DTD DocBook XML V4.1.2//EN"
-   "docbook/docbookx.dtd">
+   "/usr/share/sgml/docbook/dtd/xml/4.1.2/docbookx.dtd"
+   >
 <refentry id='greed.6'>
 <refmeta>
 <refentrytitle>greed</refentrytitle>
@@ -79,7 +80,7 @@
 <refsect1 id='files'><title>FILES</title>
 <variablelist remap='TP'>
 <varlistentry>
-<term>/usr/lib/games/greed.hs</term>
+<term>/var/games/greed/greed.hs</term>
 <listitem>
 <para>Default location of Greed high scores.</para>
 </listitem>
